docs/requirements-docs.txt

```diff
@@ -13,3 +13,4 @@ sphinxcontrib-jsmath==1.0.1
 sphinxcontrib-qthelp==1.0.3
 sphinxcontrib-serializinghtml==1.1.5
 sphinxemoji==0.2.0
+recommonmark==0.7.1
```

Comment:
Cannot install on Python 3.7 and is deprecated: https://github.com/readthedocs/recommonmark?tab=readme-ov-file#recommonmark

Does the recommended [MyST](https://github.com/executablebooks/MyST-Parser) work?

Reply:
It seems working for me with Python 3.10. Do we really need to support Python 3.7 for the docs only, considering many packages started deprecating the support of 3.7?

> Does the recommended [MyST](https://github.com/executablebooks/MyST-Parser) work?
MyST works as well, with a small cross-reference issue, where `smoke tests` fails to cross-reference the title `### Testing` due to the a problem with MyST, which can be ignored.